### ISSUE TITLE
Upgrade to Electron v11 for native Apple Silicon Support

### ISSUE THREAD

**Issue by OskarEichler** [meetfranz/franz#1992]
Hey guys,

Would be awesome if you can upgrade Electron to v11 and start supporting Apple silicon natively!

All the best,
Oskar

**Comment by OskarEichler** [comment]
Yeah it's a bit sad to see how inactive the Devs are here on Github. Franz is awesome and I'm supporting it with a Professional Plan, but them lacking behind on the development opens doors for knock-offs like Ferdi to certainly become a consideration. 

I really hope that they put out some great updates and communicate a bit more to their users in the future!

**Comment by lukegiuff** [comment]
I've made the move to Rambox which seems to be running much better on the M1 chip with seemingly native support. They also offer everything I had with Franz for free... shattered that these devs never did anything here.

**Comment by niebloomj** [comment]
Weird when a service I paid for has such inactive development...

**Comment by melvinchia** [comment]
the trail ends with a link to Ferdi on ARM/M1, but no updates or release of Franz for ARM/M1?

**Comment by sandrooco** [comment]
Any updates? I really don't want to migrate to anything else but no native M1 support is problematic for such a resource hungry app..

**Comment by niebloomj** [comment]
Make the migration, it's worth it. The dev's have killed this project unfortunately.

**Comment by sandrooco** [comment]
> Make the migration, it's worth it. The dev's have killed this project unfortunately.

WDYM they killed it? Officially?
What did you migrate to?

**Comment by niebloomj** [comment]
No response or development from the team in this long means it's been killed to me.

I switched to Ferdi https://github.com/getferdi/ferdi/

**Comment by sandrooco** [comment]
> No response or development from the team in this long means it's been killed to me.
> 
> I switched to Ferdi https://github.com/getferdi/ferdi/

It's weird that there's an active 5.8.0 branch but no response in the issues.. I hope they step up soon.

**Comment by niebloomj** [comment]
Its been over a year of an inactive development team IIRC...
